webtester-core/src/main/java/info/novatec/testit/webtester/pageobjects/TextField.java

```diff
@@ -44,7 +41,8 @@
     private static final String APPEND_TEXT = "changed text from '{}' to '{}' by trying to append '{}'";
 
     private static final String INPUT_TAG = "input";
-    private static final Set<String> VALID_TYPES = Sets.newHashSet("", "text", "password", "number");
+    private static final Set<String> INVALID_TYPES = Sets.newHashSet("button", "checkbox", "hidden",
+                "image", "radio", "range", "reset", "submit");
```

Comment:
From a HTML perspective it is the default.  A simple `<input />` will be rendered as a text field.
Browsers which haven't implemented a HTML 5 component, render them as text fields as well.

So if you produce an invalid HTML `<input type="foo" />` the browser will render and treat it as a text field.